test/mockserver/mockspanner.ts

```diff
@@ -633,6 +634,7 @@ export class MockSpanner {
         call.end();
       })
       .catch(err => {
+        call.sendMetadata(new Metadata());
```

Comment:
There are lots of other emit `error` event in the mock spanner file. Do we need to do this at all places ?

Reply:
I don't think so. My goal here is to make tests pass to allow proto updates to go through; if the Spanner teams wants to refactor the mock gRPC server used in unit tests, you can do so separately :)